includes/wc-deprecated-functions.php

```diff
@@ -14,6 +14,12 @@
 	exit;
 }
 
+include_once( 'abstracts/abstract-wc-deprecated-hooks.php' );
+include_once( 'class-wc-deprecated-action-hooks.php' );
```

Comment:
Should there also be a `include_once( 'class-wc-deprecated-filter-hooks.php' );` here?